### ISSUE TITLE
Improve direct HTML reporter (resurrect FBD reporter page)

### ISSUE THREAD

**Issue by davelab6** [googlefonts/fontbakery#3329]
In https://github.com/googlefonts/Unified-Font-Repository/issues/2 I discovered that the fontbakery HTML reporter now takes the markdown reporter and HTML'ifies it, which doesn't work well. 

The markdown reporter is made for posting FB reports to github issues/PRs, and is constrained by the limitations of not having a full web page environment. The direct HTML reporter is meant to be able to take advantage of that full environment. 

@graphicore worked on such a better HTML reporter page design for Font Bakery Dashboard, so I think the immediate next step on this issue is to resurrect that.

**Comment by felipesanches** [comment]
@graphicore please let me know if you have access to the old HTML templates and I will then be able to work on integrating them into FontBakery.

**Comment by davelab6** [comment]
They are in the fbd repo I assume

**Comment by graphicore** [comment]
It's in the FBD repo, but it's basically a web app, that gets check results live streamed via a websocket connection. I doubt it's going to be very straight forward to put it into a server-side rendered thing using Python (Node would be easier, as it could run the JS and use a DOM-API implementation to create the HTML). Further, the web-app part is very me-idiomatic. It would be an option to put the data and the JS directly in the HTML file, but it would basically be an empty html-page with data and Javascript in the header.

**Comment by graphicore** [comment]
On the other hand, rendering the HTML with new code in Python and putting just the finished controls into inline Javascript shouldn't be such a big deal, as all the live-updating stuff can go away.

**Comment by graphicore** [comment]
We can try to get the DOM the original code creates and make templates out of it, then we can re-use the CSS.

**Comment by m4rc1e** [comment]
It would be great to improve the html reporter. Just viewed this https://usted.github.io/Albert-Sans/fontbakery-report.html and it isn't so clear. Imo, it would be great if the html reporter is similar to the github MD reporter.

**Comment by felipesanches** [comment]
yeah... the HTML is not looking the same as the github MD reported, but it should. The reason for the difference must be some different level of support for markdown+HTML5 on different libraries.
